CONTRIBUTING.md

```diff
@@ -224,6 +225,54 @@ GitHub has a [nice introduction](https://help.github.com/articles/github-flow/)
 <img align="center" src="https://i.imgur.com/s8yELfK.png" alt="Example-Contribution" width="800"/>
 
 
+## Updating the schema
+
+Portions of the BIDS specification are defined using YAML files, in order to make the specification machine-readable.
+Currently, the only portion of the specification that relies on this schema is the Entity Table, but any changes to the specification should be mirrored in the schema.
+
+### The format of the schema
+
+The schema reflects the files and objects in the specification, as well as associations between these objects.
+Here is a list of the files and subfolders of the schema, with a description for each:
```

Comment:
What is the order here?  At first I thought it was alphabetical, but it is not, and the first item listed is (as far as I understand) one of the least important YAML files.

Can I suggest reordering so that they are in some order that reflects how you might have to sift through them if you are a BIDS contributor making a change.

Then maybe change this:

```suggestion
Here is a list of the files and subfolders of the schema, roughly in order of importance:
```

(No need to say that you're going to give a description... that's clear).